config/samples/custom-rules/openshift-virtualization/kubevirt-nonroot-feature-gate-is-enabled.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: compliance.openshift.io/v1alpha1
+kind: CustomRule
+metadata:
+  name: kubevirt-nonroot-feature-gate-is-enabled
+  namespace: openshift-compliance
+spec:
+  title: "KubeVirt nonRoot Feature Gate Must Be Enabled"
+  id: kubevirt_nonroot_feature_gate_is_enabled
+  description: "Unauthorized access to a root account without restrictions implemented by the nonRoot feature introduces the risk of unintended or unauthorized access to privilege elevation and the ability to perform administrative tasks."
```

Nitpick: format the description with `|-`:

```suggestion
  description: |-
    Unauthorized access to a root account without restrictions implemented by the nonRoot feature introduces 
    the risk of unintended or unauthorized access to privilege elevation and the ability to perform
    administrative tasks.
```